.changeset/breezy-feet-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+'@envelop/core': patch
```

Comment:
I think this should be a minor. It actually adds the possibility to handle incremental errors, not sure this should be considered  a bug fix :-)

```suggestion
'@envelop/core': minor
```

Reply:
gotcha, wasn't sure if defer/stream stuff was meant to be fully supported by all plugins yet